packages/ui/src/ui/components/PreloaderErrors/GeneralErrorRegion.js

```diff
@@ -12,7 +12,9 @@ import {showErrorPopup} from '../../utils/utils';
 
 const b = block('preloader');
 
-export function GeneralErrorRegion({message}) {
+export function GeneralErrorRegion({
+    message = 'Oops! Something went wrong. If problem persists please report it via Bug Reporter.',
```

Comment:
I would put the above into a constant. DEFAULT_ERROR_TEXT = 'Oops... '. It would be a bit cleaner. The text is still long

Reply:
Until the text is reused we don't need to extract the text to a separate value.